data/json/items/gun/3006.json

```diff
@@ -90,12 +91,14 @@
     "weight": "3910 g",
     "volume": "2840 ml",
     "longest_side": "1102 mm",
+    "barrel_length": "610 mm",
     "price": 80000,
     "price_postapoc": 3250,
     "to_hit": -1,
     "material": [ "steel", "wood" ],
     "color": "brown",
     "ammo": [ "3006" ],
+    "ranged_damage": { "damage_type": "bullet", "amount": 8 },
```

Comment:
Should this have both barrel and damage?

Reply:
Oops, forgot to axe that when I was doing the barrel-length updates. thanks for pointing it out.